spring-cloud-gateway-core/src/test/java/org/springframework/cloud/gateway/test/websocket/WebSocketIntegrationTests.java

```diff
@@ -315,9 +316,13 @@ public void customHeader() throws Exception {
 
 		@Bean
 		public RouteLocator wsRouteLocator(RouteLocatorBuilder builder) {
```

Comment:
Please leave the existing tests and add a new one. We don't want to regress existing functionality,

Reply:
I have made changes according to your demands/requests.
Please check it out that whether or not we need to ask/see further improvement.